fixed.vars

```diff
@@ -5,17 +5,17 @@ LODESTAR_FIXED_VARS="--dataDir /data/lodestar --execution.urls http://127.0.0.1:
 LODESTAR_VAL_FIXED_VARS="--dataDir /data/lodestar"
 
 # Data dir will be mounted in /data/nethermind
-NETHERMIND_FIXED_VARS="--datadir /data/nethermind --Network.DiscoveryPort=30303 --Network.P2PPort=30303 --Merge.Enabled=true --Init.DiagnosticMode=None --JsonRpc.Enabled=true --JsonRpc.Host=0.0.0.0 --JsonRpc.AdditionalRpcUrls \"http://localhost:8545|http;ws|net;eth;subscribe;engine;web3;client|no-auth,http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client\" --JsonRpc.JwtSecretFile /data/jwtsecret"
+NETHERMIND_FIXED_VARS="--datadir /data/nethermind --Sync.SnapSync=true --Merge.Enabled=true --Init.DiagnosticMode=None --JsonRpc.Enabled=true --JsonRpc.Host=0.0.0.0 --JsonRpc.AdditionalRpcUrls \"http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client\" --JsonRpc.JwtSecretFile /data/jwtsecret"
 
 # Data dir will be mounted in /data/geth
-GETH_FIXED_VARS="--datadir /data/geth --authrpc.jwtsecret /data/jwtsecret --http --http.api engine,net,eth,web3 --http.port 8545 --allow-insecure-unlock --http.addr 0.0.0.0 --http.corsdomain \"*\" --http.vhosts \"*\" --authrpc.port=8551"
+GETH_FIXED_VARS="--datadir /data/geth --authrpc.jwtsecret /data/jwtsecret --http --authrpc.addr 0.0.0.0 --authrpc.corsdomain \"*\" --authrpc.vhosts \"*\" --authrpc.addr 0.0.0.0 --authrpc.vhosts \"*\" --authrpc.port=8551"
```

Comment:
Geth is complaining about the flag `--authrpc.corsdomain \"*\"`. I've looked in the docs and could only find `--http.corsdomain \"*\"`. This fixed it, but unsure if the need for it on authrpc port is needed/or exists?

Reply:
Used the docs here: https://geth.ethereum.org/docs/interface/command-line-options
It's updated only to v1.10.19 and not the current v1.10.23 though FYI

Reply:
let me see because i am running geth with this locally

Reply:
yes you are correct! its http.cors domain!